<?xml version="1.0"?>
<launch>
  <arg name="link_name" />
  <arg name="frame_id" />

  <node
    name="pose_int_marker_$(arg frame_id)"
    pkg="rcprg_gazebo_utils" type="gazebo_publish_ros_tf_object" output="screen">
    <param name="link_name" value="$(arg link_name)" />
    <param name="frame_id" value="$(arg frame_id)" />
  </node>
</launch>
